docs/getting-started-guides/rkt/index.md

```diff
@@ -70,110 +65,147 @@ $ cat <<EOF >/etc/rkt/net.d/k8s_flannel_example.conf
 EOF
 ```
 
-For more information on flannel configuration, please read [CNI/flannel README](https://github.com/containernetworking/cni/blob/master/Documentation/flannel.md).
+For more information on flannel configuration, see the [CNI/flannel README](https://github.com/containernetworking/cni/blob/master/Documentation/flannel.md).
 
-##### Use Google Compute Engine (GCE) network
+#### Google Compute Engine (GCE) network
 
-Each VM on GCE has an additional 256 IP addresses routed to it, so it is possible to forego flannel in smaller clusters.
-This can most easily be done by using the builtin kubenet plugin, by setting the kubelet flag `--network-plugin=kubenet`.
+Each VM on GCE has an additional 256 IP addresses routed to it, so it is possible to forego flannel in smaller clusters. This can most easily be done by setting the kubelet option `--network-plugin=kubenet`.
 
-### Launch a local cluster
+### Launch a local Kubernetes cluster with the rkt runtime
 
-To use rkt as the container runtime, we need to supply the following flags to kubelet:
+To use rkt as the container runtime in a local Kubernetes cluster, supply the following flags to the kubelet:
 
-- `--container-runtime=rkt` chooses the container runtime to use.
-- `--rkt-api-endpoint=HOST:PORT` sets the endpoint of the rkt API service.
-  Leave empty to use the default one (`localhost:15441`).
-- `--rkt-path=$PATH_TO_RKT_BINARY` sets the path of rkt binary.
-  Leave empty to use the first rkt in $PATH.
-- `--rkt-stage1-image` sets the name of the stage1 image, e.g. coreos.com/rkt/stage1-coreos.
-  Leave empty to use the default stage1 image in the rkt's configuration.
+* `--container-runtime=rkt` Set the node's container runtime to rkt.
+* `--rkt-api-endpoint=HOST:PORT` Set the endpoint of the rkt API service. Default: `localhost:15441`.
+* `--rkt-path=PATH_TO_RKT_BINARY` Set the path of the rkt binary. Optional. If empty, look for `rkt` in `$PATH`.
+* `--rkt-stage1-image=STAGE1` Set the name of the stage1 image, e.g. `coreos.com/rkt/stage1-coreos`. Optional. If not set, the default Linux kernel software isolation stage1 is used.
 
-If you are using the [hack/local-up-cluster.sh](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/hack/local-up-cluster.sh) script to launch the local cluster, then you can edit the environment variable `CONTAINER_RUNTIME`, `RKT_PATH` and `RKT_STAGE1_IMAGE` to set these flags, the `RKT_PATH` and `RKT_STAGE1_IMAGE` are optional if you have `rkt` in your `$PATH` with appropriate configuration.
+If you are using the [hack/local-up-cluster.sh](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/hack/local-up-cluster.sh) script to launch the cluster, you can edit the environment variables `CONTAINER_RUNTIME`, `RKT_PATH`, and `RKT_STAGE1_IMAGE` to set these flags. `RKT_PATH` and `RKT_STAGE1_IMAGE` are optional if `rkt` is in `$PATH` with appropriate configuration.
```

Comment:
I *think* these {{tags}} will be expanded in production to yield, e.g., 
`https://github.com/kubernetes/kubernetes/tree/release-1.3/hack/local-up-cluster.sh`

Could a maintainer advise if these rewrite tags are still supported and/or preferred?